test/stability/testDivide/testDivide.go

```diff
@@ -394,93 +398,106 @@ func generatePacket(pkt *packet.Packet, context flow.UserContext) {
 		// Generate identical packets
 		udp.DstPort = packet.SwapBytesUint16(dstPort1)
 	}
-	ipv4.HdrChecksum = packet.SwapBytesUint16(packet.CalculateIPv4Checksum(ipv4))
-	udp.DgramCksum = packet.SwapBytesUint16(packet.CalculateIPv4UDPChecksum(ipv4, udp, pkt.Data))
-	fixMACAddrs(pkt, context)
-	atomic.AddUint64(&count, 1)
+
+	// Put a unique non-zero value
```

Comment:
you tell "put a unique", but in fact put always 2 below, comment conflicts with code, change it.

Reply:
No. I place unique values below. It is in same logic.